framework/src/Volo.Abp.AspNetCore.Components.WebAssembly/Volo/Abp/AspNetCore/Components/WebAssembly/AbpBlazorClientHttpMessageHandler.cs

```diff
@@ -38,5 +54,22 @@ private async Task SetLanguageAsync(HttpRequestMessage request, CancellationToke
                 request.Headers.AcceptLanguage.Add(new StringWithQualityHeaderValue(selectedLanguage));
             }
         }
+
+        private async Task SetAntiForgeryTokenAsync(HttpRequestMessage request)
+        {
+            var selfUri = new Uri(_navigationManager.Uri);
+
+            if (request.Method == HttpMethod.Get || request.Method == HttpMethod.Head || request.RequestUri.Host != selfUri.Host || request.RequestUri.Port != selfUri.Port)
```

Comment:
* Align with the backend: https://github.com/abpframework/abp/blob/dev/framework/src/Volo.Abp.AspNetCore.Mvc/Volo/Abp/AspNetCore/Mvc/AntiForgery/AbpAntiForgeryOptions.cs#L66
* Refactor suggestion: Separate `selfUri` control to another if condition to make it clearer.